sdk/core/Azure.Core/src/Pipeline/Internal/RequestActivityPolicy.cs

```diff
@@ -105,10 +103,15 @@ private async ValueTask ProcessAsync(HttpMessage message, ReadOnlyMemory<HttpPip
             }
             catch (Exception e)
             {
+#if NETCOREAPP2_1 // Tracing is disabled in netcoreapp2.1
+#else
                 scope.Failed(e);
```

Comment:
this one should stay on netcoreapp 2.1 - we should still support distributed tracing via DiagnosticSource

Reply:
There are a few Activity methods that are not available in netcoreapp2.1, as well as no support for ActivityKind and ActivityTagsCollection. I restored the DiagnosticScope.ActivityKind enum, but I don't believe there was anything making up for the missing methods/activitytagscollection before, I think it was just a no-op via them not existing when looking via reflection.